Baby Monitor/Source Files/Services/MediaPlayer/NodeCapture.swift

```diff
@@ -38,19 +40,12 @@ final class AudioKitNodeCapture: NSObject {
             return
         }
         node?.avAudioUnitOrNode.installTap(onBus: 0, bufferSize: AKSettings.bufferLength.samplesCount, format: node?.avAudioUnitOrNode.inputFormat(forBus: 0)) { [weak self] buffer, _ in
-           self?.bufferQueue.async {
-               guard let self = self else { return }
-               let samplesLeft = self.internalAudioBuffer.frameCapacity - self.internalAudioBuffer.frameLength
-               if buffer.frameLength < samplesLeft {
-                   self.internalAudioBuffer.copy(from: buffer)
-               } else {
-                   self.bufferReadableSubject.onNext(self.internalAudioBuffer.copy() as! AVAudioPCMBuffer)
-                   self.internalAudioBuffer = AVAudioPCMBuffer(pcmFormat: self.bufferFormat, frameCapacity: self.bufferSize)!
-                   self.internalAudioBuffer.copy(from: buffer)
-               }
-           }
+            self?.bufferQueue.async {
+                guard let self = self else { return }
+                let convertionResult = self.convertBufferIfNeeded(buffer)
```

Comment:
```suggestion
                let conversionResult = self.convertBufferIfNeeded(buffer)
```

Reply:
oh gosh 😄 I haven't realized I misspelled this all my life